fjord/feedback/tests/test_models.py

```diff
@@ -322,6 +324,8 @@ def test_purge(self):
         eq_(ResponseContext.objects.filter(
             opinion__created__gte=cutoff).count(),
             5)
+        eq_(pre_journal_count+1,
+            Record.objects.filter(action='purge_data').count())
```

Comment:
We might want to explicitly test the msg for the journal entry.

Also, interestingly, this is missing a ResponseRemoteTroubleshootingInfo portion. I'll make a note to add that after this lands.